src/main/java/Main.java

```diff
@@ -0,0 +1,60 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Main {
```

Comment:
In the content of the task, I told u that u should have minimum maps of 5 elements. In second task u have only 2 or 3 owners with thier list. You should try describe the code better, for exmaple with some comments like //TASK1 or sth like  that